src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Value.txt

```diff
@@ -91,4 +91,25 @@ true
 
 >> Value(Date(1900, 3, 1)) - Value(Date(1900, 2, 28))
 /*SqlRunner*/ Compile Error
-1
\ No newline at end of file
+1
+
+>> Boolean("true")
```

Comment:
>>> [](http://example.com/codeflow?start=0&length=2)

Boolean(Blank())? 

Boolean(error)?

Reply:
Also, Boolean(0), Boolean(1), Boolean([ ])


Boolean(true)? Coercion should round trip this, right?

Reply:
I don't think we need to support the other Boolean overloads  - it can start to get ambiguous with coercions (Boolean("0") ... false or error?) And they have reasonable unambiguous workarounds, like  (x != 0) for number.

Reply:
Coercion is not enabled for Boolean.

What would Boolean("1") return?

Reply:
Boolean("0") should be an explicit test case here (and include a comment and have it fail) - so to be clear this was a design choice and protect against somebody helpfully flipping the coercion flag in the future.

Reply:
Added negative tests